README.pt-br.md

```diff
@@ -1434,6 +1551,277 @@ configure :production, :test do
+facilmente desativar esse comportamento (o que abrirá sua aplicação para uma
+tonelada de vulnerabilidades):
+
+```ruby
+disable :protection
+```
+
+Para pular uma única camada de defesa, passe `protection` para as opções de um
+hash:
+
+```ruby
+set :protection, :except => :path_traversal
+```
+Você também pode user um array para desativar uma lista de proteções:
+
+```ruby
+set :protection, :except => [:path_traversal, :session_hijacking]
+```
+
+Por padrão, Sinatra só vai setar a sessão baseada em proteção se `:sessions`
+estiver ativada. Veja '[Using Sessions](#using-sessions)'. Algumas vezes você
+pode querer setar a sessão "fora" da aplicação Sinatra, como no config.ru ou uma
+instância separada do `Rack::Builder`. Nesse caso você ainda pode configurar a
+sessão baseada em proteção passando a opção `:session`:
+
+```ruby
+set :protection, :session => true
+```
+
+### Configurações Disponíveis
+
+<dl>
+  <dt>absolute_redirects</dt>
+    <dd>
+      Se desativado, Sinatra permitirá redirecionamentos relativos, no entanto,
+      Sinatra deixará de estar em conformidade com a RFC 2616 (HTTP 1.1), que só
+      permite redirecionamentos absolutos.
+    </dd>
+    <dd>
+      Ative se a sua aplicação estiver rodando em um proxy reverso que não foi
+      configurado corretamente. Observe que o <tt>url</tt> helper ainda vai
+      produzir URLs absolutadas, a menos que voc&e passe <tt>false</tt> como
+      segundo parâmetro.
+    </dd>
+    <dd>Desativado por padrão</dd>
+
+  <dt>add_charset</dt>
+    <dd>
+      Tipos de Mime em <tt>content_type</tt> helper adicionará automaticamente a
+      informação de charset. Você deve adicionar ao invés de substituir essa
+      opção: <tt>settings.add_charset << "application/foobar"</tt>
+    </dd>
+
+  <dt>app_file</dt>
+    <dd>
+      Caminho do arquivo principal da aplicaçãop, usado para detectar a raiz do
+      projeto, views e pastas públicas e inline templates.
+    </dd>
+
+  <dt>bind</dt>
+    <dd>
+      Endereço IP para conectar (padrão: <tt>0.0.0.0</tt> <em>ou</em>
+      <tt>localhost</tt> se seu `ambiente` estiver setado para desenvolvimento).
+      Usado apenas para servidor embutido.
+    </dd>
+
+  <dt>default_encoding</dt>
+    <dd>
+      Codificação para o caso de desconhecido (padrão é <tt>"utf-8"</tt>).
+    </dd>
+
+  <dt>dump_errors</dt>
+    <dd>Apresenta os erros no log.</dd>
+
+  <dt>environment</dt>
+    <dd>
+      Ambiente atual. Padrão é <tt>ENV['APP_ENV']</tt>, ou
+      <tt>"development"</tt> se não estiver disponível.
+    </dd>
+
+  <dt>logging</dt>
+    <dd>Usa o logger.</dd>
+
+  <dt>lock</dt>
+    <dd>
+      Coloca um bloqueio em cada requisição, executando apenas o processamento
+      de requisição por processo concorrente do Ruby.
+    </dd>
+    <dd>
+      Ativado se sua aplicação não for thread-safe. Desativado por padrão.
+    </dd>
+
+  <dt>method_override</dt>
+    <dd>
+      Use a mágica <tt>_method</tt> para permitir adicionar/excluir formulários
```

Comment:
I think he refers to the HTTP methods here.
```suggestion
      Use a mágica <tt>_method</tt> para permitir formulários PUT/DELETE
```

Reply:
Absolutely, thanks :)